test/strategy-test.ts

```diff
@@ -162,12 +162,14 @@ export function doStrategyTest(test_case: TestableStrategy) {
 
             await fastForwardAWeek();
 
-            let harvestable = await Strategy.getHarvestable();
+            //let harvestable = await Strategy.getHarvestable();
+            let harvestable = await Strategy.getWavaxAccrued();

```

Comment:
I know we discussed but will post what I ment here. Also, assuming these serve the same purpose. 
```
let harvestable_func = Strategy.functions['getHarvestable'] ? 'getHarvestable' : 'getWavaxAccrued';
let harvestable = await Strategy.functions[harvestable_func]();
...
harvestable = await Strategy.functions[harvestable_func]();
```